cinema/views.py

```diff
@@ -19,26 +20,48 @@
     OrderSerializer,
     OrderListSerializer,
 )
+from user.permissions import IsAdminOrIfAuthenticatedReadOnly
```

Comment:
do you need this import taking into account all your viewsets use default permission?